server/server.go

```diff
@@ -455,7 +469,7 @@ func (s *Server) startNetworkListener(listener net.Listener, isUnixSocket bool,
 			if authPlugin.OnConnectionEvent == nil {
 				return nil
 			}
-			host, _, err := clientConn.PeerHost("")
+			host, _, err := clientConn.PeerHost("", false)
```

Comment:
The audit log should log the real IP. I'm afraid we need to submit another issue to fix it.

Reply:
If audit log should log the real IP, the audit log plugin running code should move after handshake and before do auth. So before handshake we cannot get the REAL IP.